README.md

```diff
@@ -0,0 +1,22 @@
+# Annotations Action
+
+GitHub action for creating annotations from JSON file
+
+## Note
+
+This repository uses npm packages from `@attest` scope on github; we are working hard to open source these packages.
```

Comment:
Can't be open sourced if this is the case?

Reply:
It needs to be public for github actions to work